app/controllers/Auth.scala

```diff
@@ -190,16 +199,21 @@ class Auth @Inject() (
     })
   }
 
+
   def authenticate = Action.async { implicit request =>
     signInForm.bindFromRequest.fold(
       bogusForm => Future.successful(BadRequest(views.html.auth.signIn(bogusForm, socialProviderRegistry))),
       signInData => {
         val credentials = Credentials(signInData.email, signInData.password)
-        credentialsProvider.authenticate(credentials).flatMap { loginInfo => 
+        pool1Service.pool1user(signInData.email).flatMap {
```

Comment:
If the function `pool1user` checks if the given email address is associated to a Pool 1 user, we should rename it to `isPool1User` or something else...